From 32b06d9482429a0da4ec9a984c6b5786a9c4c553 Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Mon, 7 Mar 2016 21:20:21 +0100
Subject: [PATCH] CMake: Add vorbisenc.c to vorbis target for MSVC builds

This exports the ABI described by the .def MSVC project files.
---
 lib/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index a682ed4..ad55c85 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -58,6 +58,7 @@ set(VORBISENC_SOURCES
 )
 
 if(MSVC)
+    list(APPEND VORBIS_SOURCES vorbisenc.c)
     list(APPEND VORBIS_SOURCES ../win32/vorbis.def)
     list(APPEND VORBISENC_SOURCES ../win32/vorbisenc.def)
     list(APPEND VORBISFILE_SOURCES ../win32/vorbisfile.def)
-- 
2.7.4

